server/fileupdater.mjs

```diff
@@ -434,3 +435,12 @@ function v12HandDropShadow(properties) {
     properties.dropShadow = true;
   }
 }
+
+function v13fixedParentDefaults(properties) {
+  if (properties.type == 'card' || properties.type == 'pile' || properties.type == null){
```

Comment:
Shouldn't need to do anything for these widget types; testing for basic widget might need to ensure that `properties` called with is actually a widget.

Reply:
I think the changes I made address this, but not sure about "testing for basic widget might need to ensure that properties called with is actually a widget."  I don't really understand that.

Reply:
`properties` object passed to the function could be an element of a `BasicWidget.faces` or `Deck.cardTypes`. Of course those don't have a `type`, so I guess they still will get filtered out the same as basic widget.